core/experiments/manager.php

```diff
@@ -72,7 +72,7 @@ public function add_feature( array $options ) {
 			'on_state_change' => null,
 		];
 
-		$allowed_options = [ 'name', 'title', 'description', 'release_status', 'default', 'mutable', 'new_site', 'on_state_change' ];
+		$allowed_options = [ 'name', 'title', 'description', 'release_status', 'default', 'new_site', 'mutable', 'on_state_change', 'dependency' ];
```

Comment:
Since you support an array of dependencies:

```suggestion
		$allowed_options = [ 'name', 'title', 'description', 'release_status', 'default', 'new_site', 'mutable', 'on_state_change', 'dependencies' ];
```